app/src/main/java/com/framgia/moviedb_31/utils/BindingUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.framgia.moviedb_31.utils;
+
+import android.databinding.BindingAdapter;
+import android.support.v7.widget.RecyclerView;
+import android.widget.ImageView;
+import com.bumptech.glide.Glide;
+import com.framgia.moviedb_31.view.homescreen.HomeScreenAdapter;
+
+public class BindingUtils {
+    @BindingAdapter({ "android:moviePoster" })
+    public static void loadMoviePoster(ImageView view, String imageUrl) {
+        Glide.with(view.getContext()).load(imageUrl).into(view);
+    }
+
+    @BindingAdapter({ "android:setAdapter" })
```

Comment:
```suggestion
    @BindingAdapter({ "adapter" })
```
nên đặt tên method là động từ còn parram là danh từ